ConwaysGameOfLife/OutputFormatter.cs

```diff
@@ -24,7 +24,7 @@ public string FormatWelcomeMessage()
             return "Welcome to Game of Life!";
         }
 
-        public string FormatUniverse()
+        public string FormatUniverse() // universe created with user input - dimensions & live cells ?? - needed??

```

Comment:
we can consider writing some tests for this formatter even though we haven't started actually outputting anything to the console

```suggestion
        public string FormatUniverse(Universe universe) // universe created with user input - dimensions & live cells ?? - needed??
```